frontend/src/components/program/picasso/Picasso.vue

```diff
@@ -140,6 +140,16 @@ export default {
       required: false,
       default: null,
     },
+
+    filterSet: {
```

Comment:
Initially assumed this is a set of filters ...
```suggestion
    isFilterSet: {
```

Reply:
Or isFiltered as suggested below